frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -133,6 +137,8 @@ export class DragElementService {
                         type: '[Vehicle] Add vehicle',
                         ...createVehicleParameters(
                             this.transferringTemplate.template,
+                            this.transferringTemplate.materialTemplates,
+                            this.transferringTemplate.personnelTemplates,
```

Comment:
See the comment below for context.

```suggestion
                            selectMaterialTemplates(getStateSnapshot(this.store)),
                            selectVehicleTemplates(getStateSnapshot(this.store)),
```